docs/auth/bitbucketServer/provider.md

```diff
@@ -39,12 +46,34 @@ The Bitbucket Server provider is a structure with two configuration keys:
 To add the provider to the frontend, add the `bitbucketServerAuthApi` reference and `SignInPage` component as shown
 in [Adding the provider to the sign-in page](../index.md#sign-in-configuration).
 
+## Adding the provider to the Backstage backend
```

Comment:
We need to add the sign in resolvers config to the section above. We also have on all the auth providers a Resolvers sub-section below like this one: https://backstage.io/docs/auth/bitbucket/provider#resolvers